jre/11/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM eclipse-temurin:11.0.20.1_1-jre-alpine
```

Comment:
Understood. Thanks.

@feeblefakie As @supl said, It looks like the `eclipse-temurin` project is active, and we might not have to have the in-house images. In this case, we can depend on the `eclipse-temurin` directly in each project. What do you think?

@supl BTW, the `eclipse-temurin` project has images for jre8 and jdk8 as well, right?

Reply:
> @supl BTW, does the `eclipse-temurin` project also have images for jre8 and jdk8?

@brfrn169 

Yes. They keep releasing Java 8.
https://adoptium.net/temurin/releases/?version=8

Regarding Docker images, they provide alias tag for Java 8 too.
For example, `8-jre-alpine` is released at https://hub.docker.com/_/eclipse-temurin/tags?page=1&name=8-jre-alpine

Reply:
@brfrn169 @supl 
I think we can directly depend on `eclipse-temurin` and see how it goes.
If we need to maintain OS-level packages, we can fall back to the current way.

Reply:
> I think we can directly depend on `eclipse-temurin` and see how it goes. So, let's do it.
> If we need to maintain OS-level packages, we can fall back to the current way.

I agree with this way!

---

Also, it might be better to run `apt-get upgrade -y` in each product's Dockerfile just in case, to keep the base image latest.

FYI: https://pythonspeed.com/articles/security-updates-in-docker/

I think we should discuss this as another improvement point.